docs/advanced.rst

```diff
@@ -391,8 +391,8 @@ Pipenv supports the usage of environment variables in values. For example::
     maya = {version="*", index="pypi"}
     records = "*"
 
-Environment variables may be specified as ``${MY_ENVAR}`` or ``$MY_ENVAR``.
-On Windows, ``%MY_ENVAR%`` is supported in addition to ``${MY_ENVAR}`` or ``$MY_ENVAR``.
+Environment variables is better specified as ``${MY_ENVAR}``.
+But they may be ``$MY_ENVAR`` or ``%MY_ENVAR%`` on Windows.
```

Comment:
I believe `$MY_ENVAR` is supported in POSIX shells as well, not just on Windows. The previous wording is correct.

Reply:
Yes $MY_ENVAR is supported in POSIX shell, I wanted to say that the last option was specific to Windows only, I'll try to change the text to reflect this better